docs/release/trg-2/trg-2-3.md

```diff
@@ -2,17 +2,16 @@
 title: TRG 2.03 - Repo structure
 ---
 
-| Status | Created     | Post-History                               |
-|--------|-------------|--------------------------------------------|
-| Active | 25-Nov-2022 | adjustments to DEPENDENCY, LICENSE, NOTICE |
-|        | 24-Nov-2022 | add FOSS/Eclipse related basic files       |
-|        | 10-Nov-2022 | Initial release                            |
+| Status | Created     | Post-History                                  |
+|--------|-------------|-----------------------------------------------|
+| Active | 06-Dec-2023 | reword Why section + highlight optional files |
+|        | 25-Nov-2022 | adjustments to DEPENDENCY, LICENSE, NOTICE    |
+|        | 24-Nov-2022 | add FOSS/Eclipse related basic files          |
+|        | 10-Nov-2022 | Initial release                               |
 
 ## Why
 
-We are part of the Eclipse Foundation which requires compliance to the Eclipse Foundation rules. This also includes open and collaborative Open Source working model.
-
-Therefore, we need to follow the Eclipse Foundation and Open Source standards.
+In order to foster consistency, collaboration and ease of navigation, adhereing to a specific open source project repository structure within Eclipse Foundation is important. The structure serves a standardized framework, facilitating efficient development by providing a clear organisation of essential project components. By adopting recommended layout of files and directories, contributors can quickly locate resources such as source code, documentation and configuration files. Moreover specified repository structure aligns with industry best practices under Eclipse Foundation governance.
```

Comment:
I don't think our repository structure is necessarily connected to open source, or the Eclipse Foundation. 
We are an Eclipse Foundation project. So we need to adhere to legal documentation requirements. But this has nothing to do with the overall structure in our repository. 

I would remove the explicit mentions of open source and the Eclipse Foundation. In my opinion, this does not add further info and some of the context and statements could even be wrong

Reply:
fixed https://github.com/eclipse-tractusx/eclipse-tractusx.github.io/pull/529/commits/d005bd69be73613fbd8ffc16c2397fb9a5e37bb4, tx!